shell/assets/translations/en-us.yaml

```diff
@@ -6044,7 +6044,7 @@ performance:
     label: Incremental Loading
     setting: You can configure the threshold above which incremental loading will be used.
     description: |-
-      When enabled, resources will appear more quickly, but it may take slightly longer to load the entire set of resources. This setting only applies to the following resources: Pods, Deployments, Cron Jobs, Daemon Sets, Jobs, Stateful Sets, Replica Sets, Replication Controllers, Workloads and Secrets.
+      When enabled, resources will appear more quickly, but it may take slightly longer to load the entire set of resources
```

Comment:
We should clarify it only applies to resources that come from kube api like pagination. Maybe for simplicity 

`This setting only applies to resources that come from the Kubernetes API`

Also applies to the manual refresh description below